aptos-move/vm-genesis/src/lib.rs

```diff
@@ -438,6 +438,7 @@ pub fn default_features() -> Vec<FeatureFlag> {
         FeatureFlag::SINGLE_SENDER_AUTHENTICATOR,
         FeatureFlag::SPONSORED_AUTOMATIC_ACCOUNT_V1_CREATION,
         FeatureFlag::FEE_PAYER_ACCOUNT_OPTIONAL,
+        FeatureFlag::CONCURRENT_TOKEN_V2,
```

Comment:
The ordering is intended?

Reply:
order is just how features are defined in the enum. any order works the same